internal/services/storage/migration/share_test.go

```diff
@@ -9,55 +9,38 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/hashicorp/go-azure-sdk/sdk/environments"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
 	storageClient "github.com/hashicorp/terraform-provider-azurerm/internal/services/storage/client"
 )
 
 func TestShareV0ToV1(t *testing.T) {
-	clouds := []azure.Environment{
-		azure.ChinaCloud,
-		azure.PublicCloud,
-		azure.USGovernmentCloud,
+	input := map[string]interface{}{
+		"id":                   "share1",
+		"name":                 "share1",
+		"resource_group_name":  "group1",
+		"storage_account_name": "account1",
+		"quota":                5120,
 	}
 
-	for _, cloud := range clouds {
-		t.Logf("[DEBUG] Testing with Cloud %q", cloud.Name)
```

Comment:
The cloud/environment is not used in this migration so these shouldn't have been in the unit test anyway